test/libsolidity/syntaxTests/nameAndTypeResolution/448_no_unused_dec_after_use.sol

```diff
@@ -4,3 +4,5 @@ contract C {
         uint a;
```

Comment:
Duplicate of scoping/scoping_activation_old.sol. The other one looks more relevantly grouped,  I'd delete this one.